data/locus_coeruleus/locus_coeruleus.yml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - r-base=4.3.1
+  - bioconductor-weberdivechalcdata=1.2.0
```

Comment:
I failed to recreate your conda env. Tried on different servers and all returned the same error.
```
done                                                                                                                                                                               
ERROR conda.core.link:_execute(945): An error occurred while installing package 'bioconda::bioconductor-weberdivechalcdata-1.2.0-r43hdfd78af_0'.
Rolling back transaction: done
class: LinkError
message:
post-link script failed for package bioconda::bioconductor-weberdivechalcdata-1.2.0-r43hdfd78af_0
location of failed script: /home/jovyan/conda_envs/locus/bin/.bioconductor-weberdivechalcdata-post-link.sh
```
can you check your yml file? Esp the data package.

Reply:
Yeah that was a version issue of the package. the version 1.4.0 should work.

Reply:
Thanks! But it seems that Bioconductor can only [install the latest package via bioconda](https://github.com/Bioconductor/Bioconda_testing/blob/devel/Bioconda_SOP.md#how-to-use-bioconductor-with-bioconda). Idk if it's sustainable for future reproduction. (To be discussed)

Reply:
Ahh I see. Tbh this package is the [data itself](https://bioconductor.org/packages/release/data/experiment/html/WeberDivechaLCdata.html). If somethings changes to the data itself then I think you want an update.

Reply:
I think in this case we should not specify the package version in case of future crash @niklasmueboe